_data/sidebar_tree.yaml

```diff
@@ -45,6 +45,8 @@ tree:
       title: Updating your brain board
     - url: /tutorials/discord
       title: How to use Discord
+    - url: /tutorials/code_submitter
+      title: How to Submit Code
```

Comment:
Nit: This could be misread as how code is submitted all the time, rather than just in a virtual context. Perhaps "Using the Code Submitter"?

Reply:
Thanks, yeah wasn't entirely happy with this title. Finding something which doesn't wrap is a challenge though!

Reply:
That fits! Thanks :+1: https://github.com/srobo/docs/pull/654/commits/f7b55159fe43f030e8caf4a92bffd84801fcb7d9